mne/preprocessing/maxwell.py

```diff
@@ -505,3 +539,38 @@ def _update_sss_info(raw, origin, int_order, ext_order, nsens):
         raw.info['proc_history'] = [proc_block]
 
     return raw
+
+
+def _overlap_projector(data_int, data_res, delta=0.02):
+    """Calculate projector for removal of subspace intersection in tSSS"""
+    # delta necessary to deal with noise when finding identical signal
+    # directions in the subspace. See the end of the Results section in [2]_
+
+    # Note that the procedure here is an updated version of [2]_ using
+    # residuals instead of internal/external spaces directly. This provides
+    # data with higher rank to work with.
```

Comment:
Why is this important? Is it more numerically stable, then? Add a comment if you can.